infra/terraform/test-org/org/locals.tf

```diff
@@ -280,6 +280,7 @@ locals {
       description = "Manages Cloud Logging and Cloud Monitoring"
       topics      = local.common_topics.ops
       owners      = ["imrannayer"]
+      groups      = ["stackdriver-committers"]
```

Comment:
This group doesn't exist in the [TGM](https://github.com/orgs/terraform-google-modules/teams) org where this repo lives.

Reply:
Hey @g-awmalik, I missed creating this. It should exist now. Feel free to assign to me if you don't have bandwidth to apply.